www/js/tripconfirm/post-trip-prompt.js

```diff
@@ -119,7 +119,20 @@ angular.module('emission.tripconfirm.posttrip.prompt', ['emission.plugin.logger'
   var displayCompletedTrip = function(notification, state, data) {
     $rootScope.displayingIncident = true;
       Logger.log("About to display completed trip from Notification");
-      $state.go("root.main.tripconfirm", notification.data);
+      $state.go("root.main.enketosurvey", {
+        form_location: "json/trip-end-survey_v6.json",
+            opts: JSON.stringify({
+              session: {
+                data_key: 'manual/confirm_survey',
+                trip_properties: {
+                  // TODO: use the right variable
+                  start_ts: trip.data.properties.start_ts,
```

Comment:
Not sure how to get to this variable, perhaps something inside `notification.data` or `data`.

Reply:
`notification.data.start_ts`. You can see that we go to the `tripconfirm` screen using params = `notification.data`
https://github.com/e-mission/e-mission-phone/blob/master/www/js/tripconfirm/post-trip-prompt.js#L122
and then we get the start and end ts in the state from the `$stateParams` directly.

Reply:
also, you may not want to use the current trip end code since it has a different screen for trip end mode/purpose versus diary mode/purpose. The trip end code shows a map and the mode/purpose choices while the diary just has popovers.

The previous incident reporting trip end code would basically redirect the user to the diary screen, which seems to be closer to what you want with your survey.
https://github.com/e-mission/e-mission-phone/blob/master/www/js/incident/post-trip-prompt.js#L133

and then in the diary if we are supposed to be dealing with an incident, we refresh. you can also got to the survey screen and everything else that you wrote should Just Work.

https://github.com/e-mission/e-mission-phone/blob/master/www/js/diary/list.js#L724